geomesa-spark/geomesa-spark-sql/src/main/scala/org/locationtech/geomesa/spark/GeoMesaSparkSQL.scala

```diff
@@ -248,8 +248,11 @@ object SparkUtils extends LazyLogging {
     case GreaterThan(attr, v)             => Some(ff.greater(ff.property(attr), ff.literal(v)))
     case LessThanOrEqual(attr, v)         => Some(ff.lessOrEqual(ff.property(attr), ff.literal(v)))
     case LessThan(attr, v)                => Some(ff.less(ff.property(attr), ff.literal(v)))
-    case EqualTo(attr, v)                 => Some(ff.equals(ff.property(attr), ff.literal(v)))
-    case In(attr, values)                 => Some(values.map(v => ff.equals(ff.property(attr), ff.literal(v))).reduce[org.opengis.filter.Filter]( (l,r) => ff.or(l,r)))
```

Comment:
Just to be really, really pedantic, can we have unit tests which cover the non-___fid____ =/in cases?